lib/puppet/provider/openldap_database/olc.rb

```diff
@@ -26,11 +26,11 @@ def self.instances
       sizelimit = nil
       dbmaxsize = nil
       timelimit = nil
-      updateref = nil
+      updateref = ''
```

Comment:
I don't understand the need to change this default value. `nil` is evaluated as an empty string in a string context, and as `false` in a boolean context, so it fits the needs.